### PR TITLE
fix: update livenessProbe endpoint of zeebe

### PR DESCRIPTION
**Pull request by hamza-m-masood**
### Which problem does the PR fix?

Related to: https://github.com/camunda/camunda-platform-helm/issues/1849
### What's in this PR?

Replicating the same livenessProbe endpoint that is used in Saas since we are now using a non-existing endpoint.
### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [ ] In the repo's root dir, run `make go.update-golden-only`.
- [ ] There is no other open [pull request](../pulls) for the same update/change.
- [ ] Tests for charts are added (if needed).
- [ ] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [ ] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Comment by hamza-m-masood** [comment]
@npepinpe Would you also like me to add a startupProbe? I think this is sufficient to address the problem in the support issue.

**Comment by npepinpe** [comment]
It seems the startup probe is already there, unless I misunderstood what you asked.

File: charts/camunda-platform/values.yaml [hunk 1]
```diff
@@ -780,7 +780,7 @@ zeebe:
     ## @param zeebe.livenessProbe.scheme defines the startup probe schema used on calling the probePath
     scheme: HTTP
     ## @param zeebe.livenessProbe.probePath defines the liveness probe route used on the app
-    probePath: /actuator/health/liveness
+    probePath: /actuator/health/readiness
```
Comment (npepinpe) [inline charts/camunda-platform/values.yaml:783]: :wrench: I would add a comment explaining this is not a mistake, and we indeed use readiness here explicitly. You can link to the issue for context.

**Comment by hamza-m-masood** [comment]
Ah sorry, yes the startupProbe is already there. I added the comment.

**Pull request merged**